packages/framer-motion/src/render/dom/DOMVisualElement.ts

```diff
@@ -54,7 +54,17 @@ export abstract class DOMVisualElement<
         }
 
         if (isMounted) {
-            checkTargetForNewValues(this, target, origin as any)
+            const newValues = checkTargetForNewValues(
+                this,
+                target,
+                origin as any
+            )
+
+            // If new values have been added, for instance named color values, transform these
+            if (transformValues && newValues) {
+                origin = transformValues(origin as any)
```

Comment:
Is there any way to avoid this cast?

Reply:
Maybe but I'm just copy pasting the existing line above and would rather just get it in